index.html

```diff
@@ -0,0 +1,533 @@
+            <h2 class="white-heading">COVID-19 made your data set worthless. Now what?</h2>
+            <span class="small-text">Matthew Beedham 20 hours ago</span>
+          </div>
+          <div class="latest">
+            <div class="latest-container">
+              <div class="latest-down">
+                <h4 class="orange-h4">Latest</h4>
+                <p class="orange-par">Say hello to SHIFT, our new publication about the future of mobility tech</p>
+              </div>
+              <div class="latest-img">
+                <img src="./images/Shift-launch-cars-evs-future-polestar-electric-cars-driving-mobility.png" alt="">
+              </div>
+            </div>
+            <ul class="latest-list">
+              <li><a href="">How you can watch BMW unveil its hotly anticipated iX3 EV next week</a></li>
+              <li><a href="">Facebook takes another stab at enabling 'free' mobile web access with its new Discover
+                  app</a></li>
+              <li><a href="">Facebook-backed Reliance Jio pilots basic grocery ordering service on WhatsApp in India</a>
+              </li>
+              <li><a href="">Google's coronavirus information site is now live</a></li>
+              <li><a href="">The hyper-compact Microlino 2.0 EV is almost too cute for words</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div class="flex-container">
+        <h2 class="section-h2">Distract</h2>
+        <div class="third-section">
+          <img src="./images/hqdefault.jpg" alt="" class="hard-fork-img main-img">
+          <div class="inside-3rd-section">
+            <h3>MOST POPULAR</h3>
+            <h2 class="white-heading">Watch Ken Block tear it up in an all-electric Rallycross car</h2>
+            <span class="small-text">Matthew Beedham 3 days ago</span>
+          </div>
+          <div class="latest">
+            <div class="latest-container">
+              <div class="latest-down">
+                <h4 class="orange-h4">Latest</h4>
+                <p class="orange-par">Try to spend Bill Gates' $116B fortune in this online shopping game</p>
+              </div>
+              <div class="latest-img">
+                <img src="./images/stonksgates.jpg" alt="">
+              </div>
+            </div>
+            <ul class="latest-list">
+              <li><a href="">How to use Twitch's new Watch Party feature to binge shows with viewers</a></li>
+              <li><a href="">Tony Hawk's Pro Skater 1 + 2 is the only good thing this hell year</a></li>
+              <li><a href="">Disney sets a date for Mulan's non-premium streaming debut</a></li>
+              <li><a href="">Baby Yoda returns in The Mandalorian Season 2 this October</a></li>
+              <li><a href="">Netflix is offering a selection of original films and shows for free</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="hard-fork flex-container">
```

Comment:
- [ ] There is a redundant empty space in the medium screen view on the right side. Please fix your content to fill the full width independent from the screen size.

![image](https://user-images.githubusercontent.com/60184961/93029241-67dbcb00-f622-11ea-8dc3-9ee5813c9c1e.png)